src/pykx/pandas_api/pandas_meta.py

```diff
@@ -210,6 +210,55 @@ def abs(self, numeric_only=False):
             tab = _get_numeric_only_subtable(self)
         return q.abs(tab)
 
+    @api_return
+    def isin(self, values):
+        tab = self
+        key_table = 'KeyedTable' in str(type(tab))
+        key_value = 'KeyedTable' in str(type(values))
+        n_rows = 0
+        false_dataframe_f = q("""{u:(cols x);
+                            v:(count[u],count[x])#0b;
+                            flip u!v}""")
+        if key_value and not key_table:
+            return false_dataframe_f(tab)
+        if key_table:
+            kcols = q.key(tab)
+            if key_value:
+                n_rows, tab = q("""{n_rows:max 0, count[x]-
+                                            count rows:(key y) inter key x;
+                                            (n_rows;
+                                               x each rows)}""", tab, values)
+                values = q.value(values)
+            else:
+                tab = q.value(tab)
+        dic_value, is_tab = q("""{$[98h = type x;
+                                    (flip x; 1b);
+                                    (x; 0b)]}""", values)
+        if key_table and not key_value and is_tab:
+            ftable = false_dataframe_f(tab)
+        else:
+            ftable = q("""{ [table; values; is_tab; n_rows]
+                        flip (cols table)!
+                        {[col_name; tab; values; v_is_tab; n_rows]
+                            col: tab col_name;
+                            ltype: .Q.ty col;
+                            values: $[99h~type values; values col_name; values];
+                            $[v_is_tab or ltype=" "; ;
+                                    values@:where (lower ltype) = .Q.t abs type each values];
```

Comment:
Fix indentation in this line.

Reply:
Idents programming with strings are complicated, I have fix them so they look better. However, if I understood it correctly, if you put the full window you will see that these ident problems disappear.